site/content/user-guide/amqp-adapter.md

```diff
@@ -0,0 +1,175 @@
++++
+title = "AMQP Adapter"
+weight = 215
++++
+
+The AMQP protocol adapter allows clients (devices or gateway components) supporting the AMQP 1.0 protocol to publish telemetry messages and events to Eclipse Hono&trade;'s Telemetry and Event endpoints. The AMQP adapter supports uploading telemetry messages and events using the `AT_LEAST_ONCE` delivery semantics. It also supports uploading telemetry messages using the `AT_MOST_ONCE` (fire and forget) delivery semantics. With `AT_MOST_ONCE` delivery, the client device receives the *ACCEPTED* disposition frame from the adapter (even if the message cannot be delivered to the downstream consumer peer). This is because with `AT_MOST_ONCE`, the client has indicated that it is not interested to know what happens to the message once it has being sent. The `AT_MOST_ONCE` delivery semantics is **not supported** for events because with events, clients MUST be notified regardless of weather the message was delivered to the downstream consumer or not (in the case of a failure).
```

Comment:
I think you should try not to be too technical in this introductory paragraph. In particular, you might want to just explain that there are two ways to publish telemetry that differ in quality of service. Most readers will probably have a hard time following you through explanation of AMQP 1.0 protocol specifics at this point. So, can you try to be less technical while still explaining the different QoS?

Reply:
yes, make sense :+1: